src/main/java/com/onevizion/scmdb/SqlScriptExecutor.java

```diff
@@ -85,6 +92,9 @@ public int execute(SqlScript script) {
         } catch (SQLException e) {
             logger.error("Error during connection DB.", e);
             return SCRIPT_EXIT_CODE_ERROR;
+        } catch (IOException e) {
+            logger.error("Script OutputStream", e);
```

Comment:
change message to `Can't close OutputStream`

Reply:
Done